grails-app/assets/javascripts/streama/controllers/admin-fileManager-ctrl.js

```diff
@@ -35,6 +37,36 @@ angular.module('streama').controller('adminFileManagerCtrl', ['$scope', 'apiServ
 		})
 	};
 
+	$scope.removeMultipleFiles = function() {
+	  if($scope.deletionBulk.length > 0) {
+      var confirmText = "This will delete all selected Files. Do you want to proceed?";
+      alertify.set({ buttonReverse: true, labels: {ok: "Yes", cancel : "Cancel"}});
+      alertify.confirm(confirmText, function (confirmed) {
+        if(confirmed){
+          var bulk = JSON.stringify($scope.deletionBulk);
+          apiService.video.removeMultipleFilesFromDisk(bulk).success(function () {
+            $scope.deletionBulk.forEach(file => {
+              _.remove($scope.files, {id: file.id});
+              _.remove($scope.files, {path: file.path});
+            });
+            deletionBulk = [];
+            alertify.success('Files deleted.');
+          });
+        }
+      });
+	  }
+	};
+
+	$scope.addToDeletionBulk = function($event, file) {
+	  if($event.target.checked) {
```

Comment:
interesting, we usually check if the element is in the list, and if so, remove it, otherwise add it, but I like this approach, too!